server/src/weaverbird/backends/pypika_translator/operators.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum, auto
+
+
+class RegexOp(Enum):
+    REGEXP = auto()
+    SIMILAR_TO = auto()
+    CONTAINS = auto()
```

Comment:
`auto` returns integers. If at some point we want to configure connectors, maybe it'd be nice to have strings here ? Because right now, `RegexOp.REGEXP == FromDateOp.DATE_FORMAT`, so we might mix things up without being caught by mypy if we're not careful